### PR TITLE
docs: update the wnd docs with new parameters

### PR DESCRIPTION
**Pull request by jvillafanez**
## Description
Updating docs depending on https://github.com/owncloud/windows_network_drive/pull/475

## Related Issue
No opened issue. Related to https://github.com/owncloud/windows_network_drive/pull/475

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

@mmattel I hope I haven't broken any formatting. Feel free to take over the PR.

### CODE REVIEW

File: config/config.apps.sample.php [hunk 1]
```diff
@@ -700,34 +700,42 @@
...
-	'usermapping' => [
-		'type' => 'EALdapAttr',
-		'params' => ['attr' => 'userPrincipalName']
+	'server_ID1' => [
+		'ockeytab' => '<keytab-file-location>/<user-name>.keytab',
+		'ocservice' => 'HTTP/<hostname_or_FQDN>',
+		'usermapping' => ['type' => 'Noop'],
+		'ccachettl' => 60 * 60 * 9,
+	],
+		'server_ID2' => [
```
Comment (phil-davis) [inline config/config.apps.sample.php:721]:
```suggestion
	'server_ID2' => [
```

**Review by phil-davis — APPROVED** [review]

**Comment by mmattel** [comment]
The list does not show up as expected/intended.
I will find a way to make that working correctly

**Comment by mmattel** [comment]
**This is how it looks after fixing the list:**

image

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
@jvillafanez when will the new WND/LDAP app covering this be available?

**Comment by jvillafanez** [comment]
It's going to be merged in the 2.4.0 release, so I guess "soon", but I don't have any date.

**Comment by jnweiger** [comment]
The new features are part of https://github.com/owncloud/windows_network_drive/releases/tag/v2.4.0-rc.3
We'll do some testing in second week of January, then we know if this RC becomes a final.

**Pull request merged**